Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Support.hs

```diff
@@ -58,23 +59,27 @@ validateIssue Issue {..} =
 data SendIssueReq = SendIssueReq
   { contactEmail :: Maybe Text,
     issue :: Issue,
-    rideBookingId :: Maybe (Id Quote)
+    rideBookingId :: Maybe (Id Quote),
+    rideId :: Maybe (Id Ride)
   }
   deriving (Generic, Show, FromJSON, ToJSON, ToSchema)
 
 validateSendIssueReq :: Validate SendIssueReq
 validateSendIssueReq SendIssueReq {..} =
   validateObject "issue" issue validateIssue
 
-type SendIssueRes = APISuccess
+newtype SendIssueRes = SendIssueRes
+  { issueId :: Maybe Text
+  }
+  deriving (Generic, ToJSON, FromJSON, ToSchema)
 
 sendIssue :: EsqDBFlow m r => Id Person.Person -> SendIssueReq -> m SendIssueRes
 sendIssue personId request = do
   runRequestValidation validateSendIssueReq request
   newIssue <- buildDBIssue personId request
   runTransaction $
```

Comment:
We don't have any validation checks here. I mean we should check, that `ride` is exist, `quote` is exist, `ride` correspond to `quote`. `booking.riderId` is the same as `personId`.
So customer can send issue to any ride, not only to his own rides